wasp/src/main/java/com/orhanobut/wasp/VolleyImageHandler.java

```diff
@@ -130,6 +132,10 @@ public void run() {
 
         @Override
         public void onErrorResponse(VolleyError error) {
+            String message = error.getMessage();
+            long delay = error.getNetworkTimeMs();
+            waspImage.logWaspImageResponseError(message, delay);
```

Comment:
Maybe we can use waspImage.logRequest(), waspImage.logSuccess(), waspImage.logError().